modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1toMObjectRelatedModelsProviderImpl.java

```diff
@@ -103,9 +103,14 @@ public void disassociateRelatedModels(
 			long primaryKey2)
 		throws PortalException {
 
+		ObjectEntry objectEntry = _objectEntryService.getObjectEntry(
+			primaryKey2);
+
 		_objectEntryService.updateObjectEntry(
 			primaryKey2,
-			HashMapBuilder.<String, Serializable>put(
+			HashMapBuilder.<String, Serializable>putAll(
+				objectEntry.getValues()
+			).put(
```

Comment:
@p-albuquerque I prefer having the logic inside the `updateObjectEntry` method due to the `_deleteFromLocalizationTable` call. 

Could you check the feasibility of this change?

Reply:
If the effort is too big, we can continue with this approach.

But It's important to remember that there could be similar issues in other areas that we haven't identified yet.

Reply:
Got it. I will try this way, it's more strong for both cases and any others possibly areas